cxx/inference.cc

```diff
@@ -69,9 +70,9 @@ void inference_hirm(std::mt19937* prng, HIRM* hirm, int iters, int timeout,
   }
 }
 
-
 void inference_gendb(std::mt19937* prng, GenDB* gendb, int iters, int timeout,
                      bool verbose) {
-  // TODO(emilyaf): Also transition entity assignments.
   inference_hirm(prng, gendb->hirm, iters, timeout, verbose);
+  gendb->transition_reference_class_and_ancestors(
```

Comment:
This will do iters iterations of hirm training, followed by a single pass of entity transitions.  I don't think that's what we want.  

Rather, we probably want to add a parameter for specifying how many iterations of normal hirm training to run between entity transitions, and then use that to control the relative frequencies.

Reply:
Oops, thank you.